themes/harper/layouts/shortcodes/ta_kudos.html

```
@@ -0,0 +1 @@
+<button class="tinylytics_kudos" data-path="{{ .Page.RelPermalink }}"></button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)